src/github.com/stellar/go-horizon/resource_account.go

```diff
@@ -62,6 +76,33 @@ func NewAccountResource(ac db.AccountRecord) AccountResource {
 	// add native balance
 	balances[len(ac.Trustlines)] = BalanceResource{Type: "native", Balance: AmountToString(ac.Balance)}
 
+	// signers
+	signers := make([]SignerResource, len(ac.Signers))
+
+	for i, s := range ac.Signers {
+		signers[i] = SignerResource{Address: s.Publickey, Weight: s.Weight}
+	}
+
+	// thresholds
+	reader := strings.NewReader(ac.Thresholds)
+	b64r := base64.NewDecoder(base64.StdEncoding, reader)
+	var xdrThresholds xdr.Thresholds
+	_, err := xdr.Unmarshal(b64r, &xdrThresholds)
```

Comment:
The above logic to decode the thresholds data should be defined as a method on `db.CoreAccountRecord`